src/task/CreatePeerConnectionTask.ts

```diff
@@ -187,7 +223,13 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
       width = cap.width as number;
       height = cap.height as number;
     }
-    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId);
+    const externalUserId = this.context.realtimeController.realtimeExternalUserIdFromAttendeeId(
+      attendeeId
+    );
+    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
+    if (!externalUserId) {
```

Comment:
Missed in the previous PR: You need to check `!== null`. Customers might use an empty string as a valid external user ID.

```
if (externalUserId !== null) {
```

Advanced: `realtimeExternalUserIdFromAttendeeId` can return `string | null | undefined`.
- `string`: both attendee ID and external user ID exists
- `null`: attendee ID exists but this attendee does not have external user ID
- `undefined`: attendee ID does not exist <-- Only in this case, we can get into this if statement.

```
if (typeof externalUserId === 'undefined') {
   ...
}
```

Reply:
@simmkyu I believe ExternalUserId is a required field that must be between 2-64 characters:
https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html#API_CreateAttendee_RequestSyntax

Reply:
Also, when the external user id is read off the wire, it is converted into either a non-empty string or null here:
https://github.com/aws/amazon-chime-sdk-js/blob/efa5f360acc20793cf3993be5099264206e3327b/src/volumeindicatoradapter/DefaultVolumeIndicatorAdapter.ts#L39